Classes/Issues/IssuesViewController.swift

```diff
@@ -504,5 +516,25 @@ IssueCommentSectionControllerDelegate {
     func didEdit(sectionController: IssueCommentSectionController) {
         
     }
+    
+    func didDelete(commentID: Int) {
+        // Optimistically delete the comment
+        deletedComments.append(commentID)
+        feed.adapter.performUpdates(animated: true)
+        
+        // Actually delete the comment now
+        client.deleteComment(owner: model.owner, repo: model.repo, commentID: commentID) { [weak self] result in
+            switch result {
+            case .error:
+                if let index = self?.deletedComments.index(of: commentID) {
+                    self?.deletedComments.remove(at: index)
+                    self?.feed.adapter.performUpdates(animated: true)
+                }
+                
+                StatusBar.showGenericError()
```

Comment:
Not for this PR, but I think we can do better regarding showing actual errors.

Reply:
Agreed

Reply:
TBD w/ new toast